### PR TITLE
Konnect instructions for Adding Certificate Authorities

### PR DESCRIPTION
**Pull request by eshih**
Update to resolve issue #3451

### Summary
Adding in Konnect specific instructions to add certificates to a Kong instance for mtls.

### Reason
#3451 

### Testing
Used the instructions for a customer. Would recommend testing by using these instructions to add certificates

https://deploy-preview-3495--kongdocs.netlify.app/

### CODE REVIEW

File: app/_hub/kong-inc/mtls-auth/index.md [hunk 1]
```diff
@@ -162,7 +164,27 @@ $ curl -sX POST https://kong:8001/ca_certificates -F cert=@cert.pem
...
+4. Copy and paste your certificate information and select **Create**
+
+You will now see your certificate listed in the **Certificates** tab.
 
+To add a certificate via curl, you are required to have:
+1. Konnect ID
+2. A generated access cookie
+
+```bash
+$ curl -sX POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```
Comment (falondarville) [inline app/_hub/kong-inc/mtls-auth/index.md:184]:
```suggestion
curl -X POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```

Removed `$` per our style guide. I haven't seen `-sX` before, did you mean `-X`? If not, I can revert it.

File: app/_hub/kong-inc/mtls-auth/index.md [hunk 2]
```diff
@@ -153,6 +153,8 @@ stored in a separate `ca_certificates` store rather than the main certificates s
 they do not require private keys. To add one, obtain a PEM-encoded copy of your CA certificate
 and POST it to `/ca_certificates`:
 
+{% navtabs %}
+{% navtab Kong Admin API %}
 ```bash
 $ curl -sX POST https://kong:8001/ca_certificates -F cert=@cert.pem
```
Comment (falondarville) [inline app/_hub/kong-inc/mtls-auth/index.md:159]:
```suggestion
curl -sX POST https://kong:8001/ca_certificates -F cert=@cert.pem
```

File: app/_hub/kong-inc/mtls-auth/index.md (hunk 1 above)
Comment (eshih) [reply app/_hub/kong-inc/mtls-auth/index.md:184]: I was just following the original docs example which has `-sX` as part of the POST command. I don't think the `-s` for quiet mode is necessary for functionality so can be removed if needed.

**Review by eshih — COMMENTED** [review]
Remove `$` per style guide

File: app/_hub/kong-inc/mtls-auth/index.md (hunk 1 above)
Comment (eshih) [reply app/_hub/kong-inc/mtls-auth/index.md:184]:
```suggestion
$ curl -X POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```

File: app/_hub/kong-inc/mtls-auth/index.md [hunk 3]
```diff
@@ -153,16 +153,38 @@ stored in a separate `ca_certificates` store rather than the main certificates s
...
+4. Copy and paste your certificate information and select **Create**
+
+You will now see your certificate listed in the **Certificates** tab.
 
+To add a certificate via curl, you are required to have:
+1. Konnect ID
+2. A generated access cookie
+
+```bash
+$ curl -X POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```
Comment (eshih) [inline app/_hub/kong-inc/mtls-auth/index.md:184]:
```suggestion
curl -X POST https:konnect.konghq.com/api/control_planes/[Konnect-ID]/ca_certificates -F cert=@testCACert.pem --cookie '[generated access cookie]'
```

**Comment by eshih** [comment]
@falondarville any additional thoughts or changes needed on this PR?

**Review by falondarville — APPROVED** [review]
LGTM, thank you so much for your contribution!

**Pull request merged**